repository/templates/repository/researchitem_detail.html

```diff
@@ -13,6 +13,28 @@
 {% if item.licence %}
 <meta name="DC.rights" content="{{ item.licence }}">
 {% endif %}
+
+
+{% endblock %}
+{% block social-meta-tags %}
+<meta property="og:site_name" content="{{settings.SITE_NAME}}"/>
+<meta property="og:title" content="{{item.title}}"/>
+<meta property="og:description" content="{{item.share_abstract}}" />
+<meta property="og:url" content="{{ request.scheme }}://{{ request.get_host }}{% url 'item' item.slug %}" />
+
+<meta name="twitter:card" content="summary">
+<meta name="twitter:site" content="{{settings.SITE_TWITTER_HANDLE}}">
+<meta name="twitter:title" content="{{item.title}}">
+<meta name="twitter:description" content="{{item.share_abstract}}">
+
+{% if item.thumbnail %}
```

Comment:
Maybe the hero image from the `images-and-featuring` branch will work better because FB/Twitter love big images?